packages/language-support/src/tests/autocompletion/databasenameCompletion.test.ts

```diff
@@ -301,15 +301,15 @@ describe('Can complete database names', () => {
         { label: 'myMovies', kind: CompletionItemKind.Value },
         { label: 'scoped.alias', kind: CompletionItemKind.Value },
         { label: 'a.b.c.d', kind: CompletionItemKind.Value },
-        { label: '$param1', kind: CompletionItemKind.Variable },
       ],
       excluded: [
         { label: 'db1', kind: CompletionItemKind.Value },
         { label: 'db2', kind: CompletionItemKind.Value },
         { label: 'movies', kind: CompletionItemKind.Value },
-        // do not suggest non-string parameters
+        // do not suggest parameters, it's clear at this point we were not after a parameter in the completions
         { label: '$param2', kind: CompletionItemKind.Variable },
         { label: '$param3', kind: CompletionItemKind.Variable },
+        { label: '$param1', kind: CompletionItemKind.Variable },
```

Comment:
The reason why this test has changed is because `ALTER ALIAS a.b.c.` is well within the symbolic alias name completion, so the completion engine doesn't try to complete with parameter rule because after all of the namespaces you can only have another namespace basically. 

I've tried other approaches and the code ends up super convoluted so I'm fine with that tradeoff in this case.

For cases like `ALTER ALIAS blablablah` we are still getting parameter completions as I would expect.